src/wallet/scriptpubkeyman.cpp

```diff
@@ -2641,13 +2641,18 @@ const WalletDescriptor DescriptorScriptPubKeyMan::GetWalletDescriptor() const
 }
 
 const std::unordered_set<CScript, SaltedSipHasher> DescriptorScriptPubKeyMan::GetScriptPubKeys() const
+{
+    return GetScriptPubKeys(0);
+}
+
+const std::unordered_set<CScript, SaltedSipHasher> DescriptorScriptPubKeyMan::GetScriptPubKeys(int32_t minimum_index) const
```

Comment:
it is returned by value anyway
```suggestion
std::unordered_set<CScript, SaltedSipHasher> DescriptorScriptPubKeyMan::GetScriptPubKeys(int32_t minimum_index) const
```

Reply:
Thanks, done with a preparatory extra refactoring commit which changes the `ScriptPubKeyMan::GetScriptPubKeys()` interface (which imposes this on us).